### PR TITLE
Add `download_button`

### LINKED ISSUE #14: Download button not working
Here our config and latest gradio version

`result_gallery = ImageSlider(label='Output', interactive=True, show_download_button=True , show_label=False, elem_id="gallery1")`

Thank you for slider

### PR DESCRIPTION
**Pull request by apolinario**
Closes #14

PS: I decided that the download icon refers to the second image in the tuple. I think that's a bit arbitrary, but I'm assuming that's the standard "output" one.

### CODE REVIEW

**Review by pngwn — APPROVED** [review]
This is great. Thanks @apolinario!

**Comment by ccharest93** [comment]
Download button flag cant be used to disable the download button now.  Also not sure if this is the PR that broke the close button, but cannot reset the slider to empty anymore.

**Comment by pngwn** [comment]
Will take a look soon!

File: frontend/shared/StaticImage.svelte [hunk 1]
```diff
@@ -58,6 +61,15 @@
 	{scale}
 	{min_width}
 >
+	<!-- svelte-ignore a11y-click-events-have-key-events -->
+	<!-- svelte-ignore a11y-no-noninteractive-element-interactions-->
+	<div class="icon-buttons">
+		{#if show_download_button && value && value[1]}
+			<DownloadLink href={value[1].url} download={value[1].orig_name || "image"}>
```
Comment (byrkbrk) [inline frontend/shared/StaticImage.svelte:68]: I think, it'd be better to give an extension to the file name "image" (e.g., "image.png")

**Pull request merged**